src/index.jsx

```diff
@@ -1,22 +1,11 @@
-/* eslint-disable import/no-extraneous-dependencies */
-import React, { Fragment } from 'react';
+import React from 'react';
 import { render } from 'react-dom';
-import CssBaseline from '@material-ui/core/CssBaseline';
-import { ThemeProvider } from '@material-ui/core/styles';
-import SchemaTable from './components/SchemaTable';
-import theme from './theme';
+import SchemaViewer from './components/SchemaViewer';
 
 const root = document.getElementById('root');
 
 function App() {
-  return (
-    <Fragment>
-      <CssBaseline />
-      <ThemeProvider theme={theme}>
-        <SchemaTable />
-      </ThemeProvider>
-    </Fragment>
-  );
+  return <SchemaViewer />;
 }
 
 render(<App />, root);
```

Comment:
Since the final end-product will be a component, should we simply define `index.jsx` like how it was defined in the react-schema-viewer component (https://github.com/taskcluster/react-schema-viewer/blob/master/src/components/index.js) by removing the rendering and just exporting the component)?
```js
export {
   SchemaViewer
};
```

Reply:
> Since the final end-product will be a component, should we simply define index.jsx like how it was defined in the react-schema-viewer component (https://github.com/taskcluster/react-schema-viewer/blob/master/src/components/index.js) by removing the rendering and just exporting the component)?

Yep, that's a good idea. Note that we can still keep `src/index.jsx`. We would just need to create a `src/components/index.js`. `yarn build` only looks inside the `src/components` directory.


We can maybe use `src/index.jsx` to test that a custom theme works. This file is usually read when we do `yarn start` rather than `yarn styleguide`.

Reply:
> Yep, that's a good idea. Note that we can still keep `src/index.jsx`. We would just need to create a `src/components/index.js`. `yarn build` only looks inside the `src/components` directory.
Applied changes! 👍

Reply:
> We can maybe use `src/index.jsx` to test that a custom theme works. This file is usually read when we do `yarn start` rather than `yarn styleguide`.

Since we're in the progress of changing the styleguide to Storybook in #39 , maybe we can add an example there as well 😄